brave/brave-core
# Added post-purchased vpn sub menu in app menu

fix https://github.com/brave/brave-browser/issues/17882

Post-purchased vpn app menu
<img width="734" alt="Screen Shot 2021-09-10 at 4 29 20 PM" src="https://user-images.githubusercontent.com/6786187/132816928-e08a74ca-775c-4e71-9684-1523f6138327.png">
<img width="713" alt="Screen Shot 2021-09-10 at 4 29 28 PM" src="https://user-images.githubusercontent.com/6786187/132816935-e448d02c-2972-4565-b7a5-4bcb3e2f1723.png">
<img width="724" alt="Screen Shot 2021-09-10 at 4 29 51 PM" src="https://user-images.githubusercontent.com/6786187/132816939-f795de0e-45a0-497a-bd6a-a9f7adf2ded7.png">
<img width="713" alt="Screen Shot 2021-09-10 at 4 29 58 PM" src="https://user-images.githubusercontent.com/6786187/132816941-8d76d79e-fdee-49a8-b578-f0f5a7df8b92.png">

Pre-purchased vpn app menu for comparison
<img width="422" alt="Screen Shot 2021-09-10 at 4 32 14 PM" src="https://user-images.githubusercontent.com/6786187/132817241-678ae341-d6f0-4373-a239-30a4bbc4c467.png">
<img width="412" alt="Screen Shot 2021-09-10 at 4 32 38 PM" src="https://user-images.githubusercontent.com/6786187/132817246-0738459b-6d75-4222-8ceb-78a0be869b2c.png">


Resolves 

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
`npm run test brave_browser_tests -- --filter=BraveBrowserCommandControllerTest*`

1. Launch brave and enable vpn flag via brave://flags and re-launch
2. Launch brave with additional command line to make brave post-purchased state 
    1. `--brave-vpn-test-credentials=aa:bb:cc:dd` - make brave as post-purchased state
    2. `--brave-vpn-simulate` - use sim vpn api instead of os vpn api
3. Go to app menu and check Brave VPN item has its submenu
4. Test connect toggle button and sub label statue
5. Test Show/Hide VPN button menu
6. Other items are not yet implemented(Send feedback, About .. and Manage ...)
